emukit/bayesian_optimization/acquisitions/log_acquisition.py

```diff
@@ -0,0 +1,39 @@
+from typing import Tuple
+
+import numpy as np
+
+from emukit.core.acquisition import Acquisition
+
+
+class LogAcquisition(Acquisition):
+    """
+    Takes the log of an acquisition function
+    """
+    def __init__(self, acquisition):
+        """
+        :param acquisition: Base acquisition function
+        """
+        self.acquisition = acquisition
+
+    def evaluate(self, x: np.ndarray) -> np.ndarray:
+        """
+        :param x: Input location
+        :return: log of original acquisition function at input location(s)
+        """
+        return np.log(self.acquisition.evaluate(x))
```

Comment:
acquisition functions might be negative. Do we want a check for this or rely on responsible users?

Reply:
I have left it for the user to give us an appropriate acquisition function and added a comment in the docstring of `LocalPenalizationBatchOptimization` telling users their acquisition must output positive values only. I'm not keen on automatically applying a positive transform to the acquisition but I could add an assert here checking the acquisition is positive.

Reply:
I agree that this can be in the responsibility of the user. A sentence in the LogAcquisition doctring might be nice which points out that LogAcquisition should only get positive acquisitions.